grpc/stream_test.go

```diff
@@ -0,0 +1,264 @@
+				var client = new grpc.Client();
+				client.load([], "../vendor/google.golang.org/grpc/test/grpc_testing/test.proto");`,
+			},
+			vuString: codeBlock{
+				code: `
+				client.connect("GRPCBIN_ADDR");
+				new grpc.Stream(client, "foo/bar")`,
+				err: `method "/foo/bar" not found in file descriptors`,
+			},
+		},
+		{
+			name: "InvalidParam",
+			initString: codeBlock{code: `
+				var client = new grpc.Client();
+				client.load([], "../vendor/google.golang.org/grpc/test/grpc_testing/test.proto");`},
+			vuString: codeBlock{
+				code: `
+				client.connect("GRPCBIN_ADDR");
+				new grpc.Stream(client, "grpc.testing.TestService/EmptyCall", { void: true })`,
+				err: `unknown param: "void"`,
+			},
+		},
+		{
+			name: "InvalidTimeoutType",
+			initString: codeBlock{code: `
+				var client = new grpc.Client();
+				client.load([], "../vendor/google.golang.org/grpc/test/grpc_testing/test.proto");`},
+			vuString: codeBlock{
+				code: `
+				client.connect("GRPCBIN_ADDR");
+				new grpc.Stream(client, "grpc.testing.TestService/EmptyCall", { timeout: true })`,
+				err: "invalid timeout value: unable to use type bool as a duration value",
+			},
+		},
+		{
+			name: "InvalidTimeout",
+			initString: codeBlock{code: `
+				var client = new grpc.Client();
+				client.load([], "../vendor/google.golang.org/grpc/test/grpc_testing/test.proto");`},
+			vuString: codeBlock{
+				code: `
+				client.connect("GRPCBIN_ADDR");
+				new grpc.Stream(client, "grpc.testing.TestService/EmptyCall", { timeout: "please" })`,
+				err: "invalid duration",
+			},
+		},
+		{
+			name: "StringTimeout",
+			initString: codeBlock{code: `
+				var client = new grpc.Client();
+				client.load([], "../vendor/google.golang.org/grpc/test/grpc_testing/test.proto");`},
+			vuString: codeBlock{
+				code: `
+				client.connect("GRPCBIN_ADDR");
+				new grpc.Stream(client, "grpc.testing.TestService/EmptyCall", { timeout: "1h42m" })`,
+			},
+		},
+		{
+			name: "FloatTimeout",
+			initString: codeBlock{code: `
+				var client = new grpc.Client();
+				client.load([], "../vendor/google.golang.org/grpc/test/grpc_testing/test.proto");`},
+			vuString: codeBlock{
+				code: `
+				client.connect("GRPCBIN_ADDR");
+				new grpc.Stream(client, "grpc.testing.TestService/EmptyCall", { timeout: 400.50 })`,
+			},
+		},
+		{
+			name: "IntegerTimeout",
+			initString: codeBlock{
+				code: `
+				var client = new grpc.Client();
+				client.load([], "../vendor/google.golang.org/grpc/test/grpc_testing/test.proto");`,
+			},
+			vuString: codeBlock{
+				code: `
+				client.connect("GRPCBIN_ADDR");
+				new grpc.Stream(client, "grpc.testing.TestService/EmptyCall", { timeout: 2000 })`,
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+
+			ts := setup(t)
+
+			m, ok := New().NewModuleInstance(ts.VU).(*ModuleInstance)
+			require.True(t, ok)
+			require.NoError(t, ts.VU.Runtime().Set("grpc", m.Exports().Named))
```

Comment:
This part could stay directly on the test setup func, we don't expect to test a different jsmodule